src/fundamental/macro-fundamental.h

```diff
@@ -200,6 +200,19 @@
                 MIN(_c, z);                             \
         })
 
+/* Returns true if the passed integer is a positive power of two */
+#define CONST_ISPOWEROF2(x)                     \
+        ((x) > 0 && ((x) & ((x) - 1)) == 0)
```

Comment:
I wonder if using popcount is faster (not that it probably matters). The ll variant could always be used as the compiler is intelligent enough to pick the smallest one.

Reply:
I think it's not that clear:

https://godbolt.org/z/5GjrTxjaT

popcount actually ends up doing a function call. The x&(x-1) thing does not